test/java/org/opendatakit/briefcase/reused/UncheckedFilesTest.java

```diff
@@ -0,0 +1,65 @@
+package org.opendatakit.briefcase.reused;
+
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.opendatakit.briefcase.reused.UncheckedFiles.createFile;
+import static org.opendatakit.briefcase.reused.UncheckedFiles.createTempDirectory;
+import static org.opendatakit.briefcase.reused.UncheckedFiles.deleteRecursive;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+
+public class UncheckedFilesTest {
+  private static Path tempDir;
+  private Path temp;
+
+
+  @Before
+  public void setUp() {
+    tempDir = createTempDirectory("briefcase");
+    temp = Paths.get(tempDir.toString() + "/test.txt");
+    createFile(temp);
+  }
+
+  @After
+  public void tearDown() {
+    deleteRecursive(tempDir);
+  }
+
+  @Test(expected = UncheckedIOException.class)
+  public void newInputStream_should_throw_exception() {
+    UncheckedFiles.delete(temp);
+    UncheckedFiles.newInputStream(Paths.get(tempDir.toString() + "/test.txt"));
+  }
+
+  @Test
+  public void closeInputStream_should_handle_null() {
```

Comment:
I think this test is redundant because it's testing what's effectively a NO_OP without side effects.

This IMO is telling us that we need to remove the new `if` block in `closeInputStream` that silently allows for `null` inputStreams to be provided.